core/templates/components/common-layout-directives/navigation-bars/top-navigation-bar.component.html

```diff
@@ -580,3 +580,89 @@
     </li>
   </ul>
 </div>
+<style>
+  .dropdn {
+    position: absolute;
+    z-index: 500;
+  }
+  @media only screen and (max-width: 335px) {
+  :lang(ar) .sinin {
+font-size: 0.5rem !important;
```

Comment:
Hey @Ykumar1415 , I am not sure using the !important flag is really recommended. Could you find an alternate solution? Thanks!